services/QuillLMS/app/controllers/teacher_infos_controller.rb

```diff
@@ -28,6 +28,20 @@ def update
     }, status: 200
   end
 
+  def request_teacher_verification
+    admin_ids = request_teacher_verification_params[:admin_ids]
+    message = request_teacher_verification_params[:message]
+
+    @teacher_info.update!(verification_status: TeacherInfo::PENDING)
```

Comment:
Take or leave, but I'd be tempted to put this `update` logic on the model rather than the controller.
```ruby
class User
  def request_teacher_verification
    update!(verification_status: TeacherInfo::PENDING)
  end
end
```
That will make it easier to add additional pathways into verification workflows in the future.